plugins/interface/wlan_intf_plugin.js

```diff
@@ -56,6 +56,7 @@ class WLANInterfacePlugin extends InterfaceBasePlugin {
 
   static async preparePlugin() {
     await platform.overrideWLANKernelModule();
+    await platform.reloadWLANKernelModule();
```

Comment:
This is confusing, `overrideWLANKernelModule()` was originally just a function to initialize kernel module, now there're 2 functions but actually only one is actually doing the job. And semantically, it feels a bit weird.

Reply:
This is because there are different needs for different platforms.
For Gold, it is 8821.ko that needs to be overriden.
For PSE, it is 8821.conf that needs to be replaced.
To avoid touching existing code, the new functionality is added in a separate function rather than changing the existing logic. Each platform just needs to implement the related part to get a desired combination of the execution paths.